core/error.rs

```diff
@@ -331,28 +316,15 @@ impl JsError {
     }
     {
       let state = JsRuntime::state_from(scope);
-      let (getter, cache) = {
-        (
-          state.source_map_getter.clone(),
-          state.source_map_cache.clone(),
-        )
-      };
-      if let Some(source_map_getter) = getter {
-        let mut cache = cache.borrow_mut();
-        for (i, frame) in frames.iter().enumerate() {
-          if let (Some(file_name), Some(line_number)) =
-            (&frame.file_name, frame.line_number)
-          {
-            if !file_name.trim_start_matches('[').starts_with("ext:") {
-              source_line = get_source_line(
-                file_name,
-                line_number,
-                &mut cache,
-                &**source_map_getter,
-              );
-              source_line_frame_index = Some(i);
-              break;
-            }
+      let mut source_mapper = state.source_mapper.borrow_mut();
+      for (i, frame) in frames.iter().enumerate() {
+        if let (Some(file_name), Some(line_number)) =
+          (&frame.file_name, frame.line_number)
+        {
+          if !file_name.trim_start_matches('[').starts_with("ext:") {
+            source_line = source_mapper.get_source_line(file_name, line_number);
```

Comment:
Heads up for the follow up PR that this line will need to be adjusted to work for extension code